sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -153,7 +150,7 @@ def _init_caches(self):
             # assuming ~100 bytes per entry (query & plan signature, key hash, 4 pointers (ordered dict), expiry time)
             # total size: 10k * 100 = 1 Mb
             maxsize=int(self.check.instance.get('seen_samples_cache_maxsize', 10000)),
-            ttl=60 * 60 / int(self.check.instance.get('samples_per_hour_per_query', 15)),
+            ttl=60 * 60 / int(self.check.instance.get('samples_per_hour_per_query', 4)),
```

Comment:
Can you update the config spec since the default value is changing?

Reply:
It looks like this is a hidden config value. I'm not sure if that's intentional or an oversight. If it's an oversight, should I add the config in this PR or a separate one?

Reply:
Could you do it in a separate one if that's the case?

Reply:
Sorry for the delay: https://github.com/DataDog/integrations-core/pull/10831